src/components/TimePicker/helpers/__test__/isNumber.spec.js

```diff
@@ -0,0 +1,12 @@
+import isNumber from '../isNumber';
+
+describe('isNumber', () => {
+    it('should return true when the value paseed is an entery number, it is not an empty string and it is not an array', () => {
```

Comment:
what you mean is integer? entery does not exists and change paseed => passed
also change this names to:
- should return true when the value passed is a positive integer number
- should return false when the value passed is not a positive integer number